docs/source/install.rst

```diff
@@ -2,94 +2,97 @@ Dask Installation
-pandas, or Tornado, respectively.  This is common for downstream library
-maintainers::
+      To install Dask with ``pip`` run the following::
 
-   python -m pip install dask                # Install only core parts of dask
+         python -m pip install "dask[complete]"    # Install everything
 
-We also maintain other dependency sets for different subsets of functionality::
+      This installs Dask, the distributed scheduler, and common dependencies
+      like pandas, Numpy, and others.
 
-   python -m pip install "dask[array]"       # Install requirements for dask array
-   python -m pip install "dask[dataframe]"   # Install requirements for dask dataframe
-   python -m pip install "dask[diagnostics]" # Install requirements for dask diagnostics
-   python -m pip install "dask[distributed]" # Install requirements for distributed dask
+      You can also install only the Dask library and no optional dependencies::
 
-We have these options so that users of the lightweight core Dask scheduler
-aren't required to download the more exotic dependencies of the collections
-(Numpy, pandas, Tornado, etc.).
+         python -m pip install dask                # Install only core parts of dask
 
-Install from Source
--------------------
+      Dask modules like ``dask.array``, ``dask.dataframe``, or
+      ``dask.distributed`` won't work until you also install NumPy, pandas, or
+      Tornado, respectively.  This is uncommon for users but more common for
+      downstream library maintainers.
 
-To install Dask from source, clone the repository from `github
-<https://github.com/dask/dask>`_::
+      We also maintain other dependency sets for different subsets of functionality::
 
-    git clone https://github.com/dask/dask.git
-    cd dask
-    python -m pip install .
+         python -m pip install "dask[array]"       # Install requirements for dask array
+         python -m pip install "dask[dataframe]"   # Install requirements for dask dataframe
+         python -m pip install "dask[diagnostics]" # Install requirements for dask diagnostics
+         python -m pip install "dask[distributed]" # Install requirements for distributed dask
+
+      We have these options so that users of the lightweight core Dask scheduler
+      aren't required to download the more exotic dependencies of the collections
+      (Numpy, pandas, Tornado, etc.).
+
+   .. tab-item:: Source
+
+      To install Dask from source, clone the repository from `github
+      <https://github.com/dask/dask>`_::
+
+         git clone https://github.com/dask/dask.git
+         cd dask
+         python -m pip install .
+
+      You can also install all dependencies as well::
 
-You can also install all dependencies as well::
+         python -m pip install ".[complete]"
 
-    python -m pip install ".[complete]"
+      You can view the list of all dependencies within the ``extras_require`` field
+      of ``setup.py``.
```

Comment:
This predates the changes in this PR, but might as well fix since it's easy 

```suggestion
      You can view the list of all dependencies within the ``project.optional-dependencies`` field
      of ``pyproject.toml``.
```